### ISSUE TITLE
Should pg_tester work in Linux?

### ISSUE THREAD

**Issue by janpipek** [Microsoft/pgtester#6]
Sorry for my ignorance but I am not able to use `pg_tester` in Linux. It fails when starting the server. I tred to manually run the same command, i.e.: 

     /usr/bin/pg_ctl start -o '-p 6433' -D /tmp/pg_tester`

And I obtain the following error message:

    could not create lock file "/var/run/postgresql/.s.PGSQL.6433.lock": Permission denied

Intuitively, I would expect something like that; an unprivileged user typically should be able to write there. So the question is - is there any way to run pg_tester without root / postgres account? Or what amendments should I make to my user environment that will not be an open security hole?

Thanks for any hints on this problem.

P.S. My colleagues are happily using pg_tester in macOS.

**Comment by dbadura** [comment]
Yes, you are able to run PostgreSQL as normal user. In `postgresql.conf` look at `unix_socket_directories` and change it to which you own. 
Look also at [PostgreSQL Docs](https://www.postgresql.org/docs/9.6/runtime-config-connection.html)
